components/NewResourceComment.js

```diff
@@ -0,0 +1,37 @@
+import { OpenAPIProvider } from "react-openapi-client";
+import React from "react";
+import { Button, Modal, Box } from "@mui/material";
+import NewResourceCommentForm from "./NewResourceCommentForm";
+
+function NewResourceComment({ resourceId }) {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const style = {
+    position: "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: 600,
+    bgcolor: "background.paper",
+    border: "2px solid #000",
+    boxShadow: 24,
+    p: 4,
+  };
+
+  return (
+    <OpenAPIProvider definition="http://localhost:3001/api-docs/v1/swagger.json">
```

Comment:
we should use relative paths to avoid changing things when deploying. Just start with /api-docs(...) I think

Reply:
Fixed as you said, with the relative path only